rdrf/rdrf/templates/rdrf_cdes/archive_modal.html

```diff
@@ -1,53 +1,78 @@
 {% load i18n %}
 {% load translate %}
 {% csrf_token %}
-{% if show_archive_button %}
```

Comment:
Sorry this file looks like it has changed so much. I was having trouble with indenting and then decided to reformat the indenting across the whole file.

Main changes that I can remember:
* Remove checks for `show_archive_button`
* Replace checks for `not_linked` with javascript logic that shows/hides content based on the selected patient and if they are linked or not
* Changes to the javascript to pass around data attributes from the triggering element to the archive modal based on the selected patient.